src/vm/i386/unixstubs.cpp

```diff
@@ -6,7 +6,11 @@
 
 extern "C"
 {
-    void ThrowControlForThread()
+    void ThrowControlForThread(
+#ifdef WIN64EXCEPTIONS        
```

Comment:
I'm sorry for not being clear. What I intended is "Should I add a patch to enable ``WIN64EXCEPTIONS`` for x86/Linux in this PR?". (That patch is not added, yet.)